core/broadleaf-profile/src/main/java/org/broadleafcommerce/profile/core/domain/ChallengeQuestionImpl.java

```diff
@@ -36,7 +36,7 @@
 @Entity
 @Inheritance(strategy = InheritanceType.JOINED)
 @Table(name = "BLC_CHALLENGE_QUESTION")
-@Cache(usage = CacheConcurrencyStrategy.READ_WRITE, region="blStandardElements")
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE, region="blProfileElements")
```

Comment:
This can probably just stay in `blStandardElements` it's hardly ever used and if we are going to change `blProfileElements` to `blCountryElements` or whatever then it doesn't really fit there.